deployment/armada-bundle/values.yaml

```diff
@@ -143,8 +150,76 @@ armada-lookout:
   prometheus:
     enabled: true
 
-  applicationConfig: {}
+  applicationConfig:
+    disableEventProcessing: true
+    eventQueue: "ArmadaLookoutEventProcessor"
+    nats:
+      Servers:
+        - "nats://stan.armada.svc.cluster.local:4222"
+      ClusterID: "test-cluster"
+      Subject: "ArmadaTest"
+    postgres:
+      connection:
+        host: postgresql.armada.svc.cluster.local
+        port: 5432
+        user: postgres
+        password: psw
+        dbname: postgres
+
+armada-lookout-migration:
+  image:
+    tag: v0.3.36
+  ingressClass: nginx
+  clusterIssuer: letsencrypt-dev
+
+  prometheus:
+    enabled: true
+
+  applicationConfig:
+    postgres:
+      maxOpenConns: 100
+      maxIdleConns: 25
+      connMaxLifetime: 30m
+      connection:
+        host: postgresql.armada.svc.cluster.local
+        port: 5432
+        user: postgres
+        password: psw
+        dbname: postgres
+
+armada-lookout-ingester:
+  image:
+    repository: gresearchdev/armada-lookout-ingester-dev
+    tag: 88ea8f0b8124c7dbbb44f7c7315c0fca13655f18
+  applicationConfig:
+    postgres:
+      maxOpenConns: 100
+      maxIdleConns: 25
+      connMaxLifetime: 30m
+      connection:
+        host: postgresql.armada.svc.cluster.local
+        port: 5432
+        user: postgres
+        password: psw
+        dbname: postgres
+        sslmode: disable
+    pulsar:
+      enabled: true
+      URL: "pulsar://pulsar-broker.armada.svc.cluster.local:6650"
+      jobsetEventsTopic: "persistent://armada/armada/events"
 
 pulsar:
+  init:
+    enabled: false
```

Comment:
Is it right that this is disabled?

Reply:
Hmmm, so you can configure this chart to use external components for Pulsar, NATS/STAN, Redis, Postgres... or use install them from this umbrella chart.
Should we in all cases initialize Pulsar? Maybe we can actually, I think the Puslar init script is idempotent

Reply:
Ah. In that case I guess we leave it disabled? Better to only initialise once.